packages/docs/src/routes/docs/deployments/cloudflare-pages/index.mdx

```diff
@@ -45,11 +45,11 @@ npm run build
 
 ## Deploy to Cloudflare Pages
 
-After installing the integration using `npm run qwik add cloudflare-pages`, the project is ready to be deployed to Cloudflare Pages. However, you will need to create a git repository and push the code to it.
```

Comment:
You don't need to create a git repository in order to deploy a Cloudflare Pages app, you can do it directly using `wrangler` (although our guide linked below doesn't provide any info about that 😓)